PnX-SI/GeoNature
# fix(CA): added get to the route

also opened the file in a new tab and fixed the flashing link when the ca is not closed

### To Test
- You need a CA (cadre d'aquisition) with a jdd (jeu de données) and an observation
- Go to the CA information page in metadata
- check that the link below the export button doesn't pop in and out
- go back one page close the ca
- go check that the link is there and that when you click it works and open on a new page
